torch_geometric/transforms/mask.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Optional, Union
+
+from torch_geometric.data import Data
+from torch_geometric.data.datapipes import functional_transform
+from torch_geometric.transforms import BaseTransform
+from torch_geometric.utils import index_to_mask, mask_to_index
+
+
+def attrs_ends_with(data: Data, suffix: str):
+    return tuple(filter(lambda s: s.endswith(suffix), data.keys))
+
+
+def update_suffix(name, old_suffix, new_suffix):
+    if name.endswith(old_suffix):
+        name = name[:-len(old_suffix)]
+
+    return name + new_suffix
+
+
+@functional_transform('index_to_mask')
+class IndexToMask(BaseTransform):
+    r"""Converts indices to a mask representation
+
+    Args:
+        attrs (str, [str], optional): If given, will only perform index to mask
+            conversion for the given attributes. If omitted, will infer the
+            attributes from the suffix `_indices`  (default: :obj:`None`)
```

Comment:
```suggestion
            attributes from the suffix :obj:`*_idx`  (default: :obj:`None`)
```
Can we name this `_idx` rather than `_indices`?

Reply:
sure! I didn't really like `indices`